src/stan-users-guide/parallelization.Rmd

```diff
@@ -28,25 +28,21 @@ law](https://en.wikipedia.org/wiki/Amdahl's_law).
 
 ## Reduce-Sum { #reduce-sum }
 
-The higher-order reduce with summation facility maps evaluation of a
-function `g: U -> real`, which returns a scalar, to a list of type
-`U[]`, `{ x1, x2, ... }`, and performs as reduction operation a sum
-over the results. For instance, for a sequence of ```x``` values of
-type ```U```, ```{ x1, x2, ... }```, we might compute the sum:
+It is often necessary in probabilistic modeling to compute the sum of
+a number of independent function evaluations. This occurs, for instance, when
+evaluating a number of conditionally independent terms in a log-likelihood.
+If `g: U -> real` is the function and `{ x1, x2, ... }` is an array of
+inputs, then that sum looks like:
 
 `g(x1) + g(x2) + ...`
 
-In probabilistic modeling this comes up when there are $N$
-conditionally independent terms in a likelihood. Because of the
-conditional independence, these terms can be computed in parallel. If
-dependencies exist between the terms, then this isn't possible. For
-instance, in evaluating the log density of a Gaussian process then
-summation of independent terms isn't applicable.
+The `reduce_sum` function is a tool for automatically parallelizing these
```

Comment:
I avoided to speak directly of `reduce_sum`, but rather speak of reduce summation. The actual implementations are then `reduce_sum` and `reduce_sum_static`. Does that logic make sense to you?

Reply:
I just added them both here. I wanted to say the function names explicitly to avoid talking about a reduce, or parallel for, or map or any of that stuff.